src/chats/components/secondaryBar/ExpandedSidebarListItem.tsx

```diff
@@ -171,7 +171,7 @@ const ExpandedSidebarListItem: React.FC<ExpandedSidebarListItemProps> = ({ roomI
 				case MessageType.CONFIGURATION_MSG:
 					return <ConfigurationMessageLabel message={lastMessageOfRoom} />;
 				default:
-					return `affiliation message to replace`;
```

Comment:
I think here we should at least handle with a console.error the case. Otherwise we will lose the informationsor missing cases